pkg/daemon/daemon.go

```diff
@@ -62,15 +66,15 @@ type Daemon struct {
 func New(
 	nodeName string,
 	namespace string,
+	stdoutToSocket bool,
 	kubeClient *kubernetes.Clientset,
 	ptpUpdate *LinuxPTPConfUpdate,
 	stopCh <-chan struct{},
 ) *Daemon {
-	RegisterMetrics(nodeName)
```

Comment:
I don't see any call for this function.
we will need this if the sidecar is not enabled no?